dtp_apis/fetch_DTP_API.py

```diff
@@ -583,6 +583,60 @@ def fetch_activity_nodes(self, url=None):
         req_url = self.DTP_CONFIG.get_api_url('get_find_elements') if not url else url
         return self.post_general_request(payload, req_url).json()
 
+    def fetch_task_nodes(self, url=None):
+        """
+        The method queries task nodes from the platform.
+
+        Parameters
+        ----------
+        url : str, optional
+            used to fetch a next page
+
+        Returns
+        ------
+        dictionary
+            JSON mapped to a dictionary. The data contain task nodes.
+        """
+
+        payload = json.dumps({
+            "query": {
+                "$domain": self.DTP_CONFIG.get_domain(),
+                "$classes": {
+                    "$contains": self.DTP_CONFIG.get_ontology_uri('task')
```

Comment:
does this work with lowercase here?
The URL is "https://www.bim2twin.eu/ontology/Core#Task"
I don't know if the lowercase will be converted correctly to uppercase.

Reply:
`task` is a string mapped to task IRI on config `<URI function="task" type="xs:anyURI">https://www.bim2twin.eu/ontology/Core#Task</URI>`